output/Form.md

```diff
@@ -103,10 +103,29 @@ title: Yerun Open Science Award form
     initiative, or would you like to initiate a new project with the financial
     resources from the award?*
 
-    We will use the money for training. We will develop an SOP for the lab. We
-    will embed new projects in this OS framework. We will try to migrate
-    previous projects when possible. We will also try to anonymize datasets to
-    avoid GDPR and make them publicly accessible -if not possible, we will use
-    OPAL and DataShield.
-
-<!--# blank field, max 400 words -->
+    Our aim is to embed all our scientific production within this workflow,
+    leveraging as much as possible on open science practices from beginning to
+    end. In order to this, we want to dig further into our understanding and
+    know-how of data access management with Opal (and the Obiba suite in
+    general), literate computing, and version control with git/GitHub. All these
+    technologies enable creating reliable reproducible workflows, but learning
+    to naviagte them is time- and effort-consuming; we need to draw our
+    knowledge from the field of software engineering and computer science, and
+    then adapt it and translate it to the needs and particularities of
+    scientific production.
+
+    At the present time, our self assessment is that we are in a maturing phase
+    of our skill-development process with these technologies. Before we are able
+    to expand our best practices outside our own group, we need to consolidate
+    our knowledge. Consequently, we believe that we need to take courses and
+    training activities.
+
+    Therefore, we would allocate the prize for training in the use of these
+    technologies. We would like to develop an expertise that can then be
+    translated into formative actions that can be exported to the rest of the
```

Comment:
"... into formative actions extensible to other lab members and the research community"

Reply:
@JesusSierralaya can you take over and implement @robertomediavilla's suggestion?